os/board/rtl8730e/src/component/soc/amebad2/fwlib/ram_hp/ameba_mipi_show.c

```diff
@@ -0,0 +1,812 @@
+#include "ameba_soc.h"
```

Comment:
Is this file an example of MIPI?
When MIPI is working, this file is unnecessary?
We have an issue on flash size (not enough flash size so that we should do flash usage optimization). So, if it's just an example, we should not include this. Are you able to move this into the apps/examples?

Is there any other file like this?

Reply:
Dear @sunghan-chang,
For MIPI example, I will create a board specific example and add macro, so that these examples file will only be built when example is enable. 
Another thing to take note is :since currently there isn't a porting layer available, so I will just extern the example similar to wificsi, and will do the relevant changes after porting layer is ready. 

One thing regarding FPS to confirm with you, i will be printing the maximum FPS(137) on the top right corner of the LCD screen based on the configuration(rgb888, framesize 480*800 etc) set in the example, is this ok? Once it is confirmed i will raise the PR again with the relevant changes.

Thank you
BR,
Zhongnuo